src/Features/Homescreen.php

```diff
@@ -134,4 +135,17 @@ public function preload_options( $options ) {
 
 		return $options;
 	}
+
+	/**
+	 * Add data to the shared component settings.
+	 *
+	 * @param array $settings Shared component settings.
+	 */
+	public function component_settings( $settings ) {
+		$allowed_statuses       = array( 'pending', 'processing', 'completed' );
+		$status_counts          = array_map( 'wc_orders_count', $allowed_statuses );
+		$settings['orderCount'] = array_sum( $status_counts );
```

Comment:
Should the allowed statuses also contain cancelled, refunded, and failed? Maybe also on hold?

Reply:
In case this factors in, there's a setting where users can define their own "actionable" order statuses so I think it would be a possibility that they could define Cancelled, Refunded, and Failed statuses as actionable to appear in the Orders panel.

By the way @pmcpinto, Are you seeing those settings when you test the new nav? They're supposed to be in *Analytics > Settings*.

Reply:
Hmm, you're right - this is too narrow a list of order statuses to check. How about we just look for any orders that aren't trashed or draft (used by the checkout block)?

Reply:
Updated in a96f21d.

Reply:
sounds good

Reply:
> By the way @pmcpinto, Are you seeing those settings when you test the new nav? They're supposed to be in Analytics > Settings.

@elizaan36 yes: https://d.pr/i/lYE8RS